common/beerocks/bcl/include/beerocks/bcl/beerocks_defines.h

```diff
@@ -26,12 +26,12 @@ namespace beerocks {
 #define BEEROCKS_PLAT_MGR_UDS "uds_platform_manager"
 #define BEEROCKS_BACKHAUL_MGR_UDS "uds_backhaul_manager"
 
-#if __GNUC__ >= 7
-#define FALLTHROUGH [[fallthrough]]
+#if __GNUC__
```

Comment:
On second inspection: the GCC fallthrough attribute was also only introduced in GCC 7, so this makes no sense. The first version was OK.

We could improve it by also supporting C++17 (which standardizes the `[[fallthrough]]` attribute):
```
#if __GNUC__ >= 7 || __cplusplus >= 201703L